rust/main/agents/scraper/src/db/message.rs

```diff
@@ -305,25 +313,27 @@ impl ScraperDb {
             return Ok(0);
         }
 
-        Insert::many(models)
-            .on_conflict(
-                OnConflict::columns([
-                    message::Column::OriginMailbox,
-                    message::Column::Origin,
-                    message::Column::Nonce,
-                ])
-                .update_columns([
-                    message::Column::TimeCreated,
-                    message::Column::Destination,
-                    message::Column::Sender,
-                    message::Column::Recipient,
-                    message::Column::MsgBody,
-                    message::Column::OriginTxId,
-                ])
-                .to_owned(),
-            )
-            .exec(&self.0)
-            .await?;
+        for chunk in models.chunks(Self::STORE_MESSAGE_CHUNK_SIZE) {
+            Insert::many(chunk.to_vec())
```

Comment:
Not sure if theres a way to prevent cloning the chunk here.

Reply:
Ok